python/ray/serve/backend_worker.py

```diff
@@ -229,6 +253,12 @@ async def invoke_single(self, request_item: Query) -> Any:
             self.error_counter.record(1, {"backend": self.backend_tag})
         finally:
             self._reset_context()
+        self.processing_latency_tracker.record(
+            (time.time() - start) * 1e3, {
```

Comment:
nit easier to read for small numbers
```suggestion
            (time.time() - start) * 1000, {
```